snmp/check.py

```diff
@@ -492,6 +492,7 @@ def submit_metric(self, name, snmp_value, forced_type, tags=[]):
             return
 
         metric_name = self.normalize(name, prefix="snmp")
+        self.log.debug("Submitting %s", metric_name)
```

Comment:
Logging every metric we submit sounds a bit too verbose to me, even for debug-level logs. What do you think?